internal/pkg/api/server/buffer/dependencies/dependencies.go

```diff
@@ -127,13 +127,16 @@ func NewServerDeps(serverCtx context.Context, envs env.Provider, logger log.Pref
 	serverWg := &sync.WaitGroup{}
 
 	// Get Storage API host
-	storageApiHost := strhelper.NormalizeHost(envs.MustGet("KBC_STORAGE_API_HOST"))
+	storageApiHost := strhelper.NormalizeHost(envs.Get("KBC_STORAGE_API_HOST"))
 	if storageApiHost == "" {
-		return nil, errors.New("KBC_STORAGE_API_HOST environment variable is not set")
+		return nil, errors.New("KBC_STORAGE_API_HOST environment variable is empty or not set")
```

Comment:
Tu teda podmienka bola, ale bol preklep v `MustGet -> Get`, super ze sa to opravilo.